contracts/staking/test/stake_test.ts

```diff
@@ -324,131 +250,135 @@ blockchainTests.resets('Stake Statuses', env => {
-            await staker.unstakeAsync(amount, new StakingRevertErrors.InsufficientBalanceError(amount, ZERO));
+            await staker.unstakeAsync(amount);
         });
     });
     describe('Simulations', () => {
-        it('Simulation from Staking Spec', async () => {
-            // Epoch 1: Stake some ZRX
-            await staker.stakeAsync(toBaseUnitAmount(4));
-            // Later in Epoch 1: User delegates and deactivates some stake
-            await staker.moveStakeAsync(
-                new StakeInfo(StakeStatus.Active),
-                new StakeInfo(StakeStatus.Inactive),
-                toBaseUnitAmount(1),
-            );
-            await staker.moveStakeAsync(
-                new StakeInfo(StakeStatus.Active),
-                new StakeInfo(StakeStatus.Delegated, poolIds[0]),
-                toBaseUnitAmount(2),
-            );
-            // Epoch 2: Status updates (no user intervention required)
-            await staker.goToNextEpochAsync();
-            // Epoch 3: Stake that has been inactive for an epoch can be withdrawn (no user intervention required)
-            await staker.goToNextEpochAsync();
-            // Later in Epoch 3: User reactivates half of their inactive stake; this becomes Active next epoch
-            await staker.moveStakeAsync(
-                new StakeInfo(StakeStatus.Inactive),
-                new StakeInfo(StakeStatus.Active),
-                toBaseUnitAmount(0.5),
-            );
-            // Later in Epoch 3: User re-delegates half of their stake from Pool 1 to Pool 2
-            await staker.moveStakeAsync(
-                new StakeInfo(StakeStatus.Delegated, poolIds[0]),
-                new StakeInfo(StakeStatus.Delegated, poolIds[1]),
-                toBaseUnitAmount(1),
-            );
-            // Epoch 4: Status updates (no user intervention required)
-            await staker.goToNextEpochAsync();
-            // Later in Epoch 4: User deactivates all active stake
-            await staker.moveStakeAsync(
-                new StakeInfo(StakeStatus.Active),
-                new StakeInfo(StakeStatus.Inactive),
-                toBaseUnitAmount(1.5),
-            );
-            // Later in Epoch 4: User withdraws all available inactive stake
-            await staker.unstakeAsync(toBaseUnitAmount(0.5));
-            // Epoch 5: Status updates (no user intervention required)
-            await staker.goToNextEpochAsync();
-            // Later in Epoch 5: User reactivates a portion of their stake
-            await staker.moveStakeAsync(
-                new StakeInfo(StakeStatus.Inactive),
-                new StakeInfo(StakeStatus.Active),
-                toBaseUnitAmount(1),
-            );
-            // Epoch 6: Status updates (no user intervention required)
-            await staker.goToNextEpochAsync();
-        });
+        // it('Simulation from Staking Spec', async () => {
```

Comment:
[reminder to uncomment this before merge]

Reply:
Do we actually still need this? I assume that part of the spec is going to change a lot with the changes from this PR.

Reply:
I'm going to leave this commented out for now. Unclear to me how we will be updating the flow in the spec. The interactions become so much simpler that I'm not even sure we need this test.